crosswalks-sssom/template/TARGET_VOCAB-codemeta-mappings.csv

```diff
@@ -1,17 +1,78 @@
-source_term,codemeta_term,type_relation,combined_mappings,comments
-repository,codeRepository,exact_match,,
-version,softwareVersion,exact_match,,
-file,downloadUrl,exact_match,,
-version,softwareVersion,exact_match,,
-note,releaseNotes,exact_match,,
-author,author,exact_match,,
-year,datePublished,part_of,$(year)+$(month), 
-month,datePublished,part_of,$(year)+$(month),
-editor,editor,exact_match,,
-license,license,exact_match,,
-publisher,publisher,exact_match,,
-abstract,description,exact_match,,
-doi,identifier,more_specific_than,,
-hal_id,identifier,more_specific_than,,
-url,url,exact_match,,
-organization,affiliation,exact_match,,
+source_term,codemeta_term,combined_mappings,comments
+,applicationCategory,,
+,applicationSubCategory,,
+,author/affiliation/identifier,,
+,author/affiliation/name,,
+,author/email,,
+,author/familyName,,
+,author/givenName,,
+,author/identifier,,
+,author/roleName,,
+,buildInstructions,,
+,citation,,
+,codeRepository,,
+,continuousIntegration,,
+,contributor/affiliation/identifier,,
+,contributor/affiliation/name,,
+,contributor/email,,
+,contributor/familyName,,
+,contributor/givenName,,
+,contributor/identifier,,
+,contributor/roleName,,
+,copyrightHolder,,
+,copyrightYear,,
+,dateCreated,,
+,dateModified,,
+,datePublished,$(year)+$(month), 
+,description,,
+,developmentStatus,,
+,downloadUrl,,
+,editor,,
+,embargoEndDate,,
+,encoding,,
+,endDate,,
+,fileFormat,,
+,fileSize,,
+,funder/name,,
+,funding,,
+,hasPart,,
+,hasSourceCode,,
+,identifier,,
+,installUrl,,
+,isAccessibleForFree,,
+,isPartOf,,
+,isSourceCodeOf,,
+,issueTracker,,
+,keywords,,
+,license,,
+,maintainer,,
+,memoryRequirements,,
+,name,,
+,operatingSystem,,
+,permissions,,
+,position,,
+,processorRequirements,,
+,producer,,
+,programmingLanguage,,
+,provider,,
+,publisher,,
+,readme,,
+,referencePublication,,
+,relatedLink,,
+,releaseNotes,,
+,review,,
+,reviewAspect,,
+,reviewBody,,
+,runtimePlatform,,
+,sameAs,,
+,softwareHelp,,
+,softwareRequirements,,
+,softwareSuggestions,,
+,softwareVersion,,
+,sponsor,,

```

Comment:
shouldn't this be consistent with funder?
have sponsor/name and sponsor/url or similar

Reply:
need to remove line 76 then, no?